draft-barnes-acme.md

```diff
@@ -1646,12 +1646,15 @@ The client serializes the validation object to UTF-8, then uses its account
 private key to sign a JWS with the serialized JSON object as its payload.  This
 JWS is NOT REQUIRED to have the "nonce" header parameter.
 
-The client will generate a self-signed certificate with the subject's
-organizationName field set to the "signature" value from the JWS, i.e., the
-base64-encoded signature value, and a subjectAlternativeName extension
-containing a single dNSName of "\<token\>.acme.invalid".  The client will then
-configure the TLS server at the domain such that when a handshake is initiated
-with the Server Name Indication extension set to "\<token\>.acme.invalid", the
+The client will compute Z, the SHA-256 of the "signature" value from the JWS.
+The hash is calculated over the base64-encoded signature string.  Z is encoded
```

Comment:
I guess technically you would need to say UTF-8 / ASCII, but this is probably clear enough.